.gitignore

```diff
@@ -1,8 +1,7 @@
 scripts/keys.py
-scripts/*json
-scripts/node_modules/
-scripts/__pycache__/keys.cpython-310.pyc
```

Comment:
You also removed `keys.cpython-310.pyc`.

Reply:
Ah I see *.pyc is already ignored

Reply:
`scripts/__pycache__/keys.cpython-310.pyc` is ignored by virtue of both `__pycache__/` and `*.pyc`